src/CommandingOfficers/AW1/BM/Grit.java

```diff
@@ -87,6 +87,7 @@ private static class SnipeAttack extends AW1BasicAbility
     @Override
     protected void enqueueUnitMods(MapMaster gameMap, ArrayList<UnitModifier> modList)
     {
+      super.enqueueUnitMods(gameMap, modList);
       modList.add(rangeMod);
       modList.add(atkMod);
```

Comment:
I'm thinking it might be a good idea to make the `AW1BasicAbility` declare this `final` in the root and call a new function it defines for its children to overload?
Ditto for the rest of the games, 'cause I apparently keep forgetting to add these.

Reply:
I could go either way I think. Making this final and creating a new override just means you'd have to remember to include the base bonuses for the description text along with any additions you make. Leaving it like this means subclasses have to remember to include the base bonuses at all with their additions.